### ISSUE TITLE
Add YAPP website to README

### ISSUE THREAD

**Issue by ani-per** [alopezlago/YetAnotherPacketParser#52]
As far as I can tell, the [YAPP demo website](https://www.quizbowlreader.com/yapp.html) is not linked anywhere in this repo. You can only find it by going to the [quizbowlreader main page](https://www.quizbowlreader.com/).

Is this intentional? If not, could the website be added to the repo main page and/or the README?

**Comment by alopezlago** [comment]
No, the site was made much later and has its own private repo. It makes sense to add a link to the demo in the readme.
